examples/automatic-speech-recognition/whisper.py

```diff
@@ -53,6 +57,30 @@
     if args.hub_token is not None:
         login(args.hub_token)
 
-    model = WhisperForConditionalGeneration.from_pretrained(args.model)
+    torch_dtype = torch.float32
+    model = WhisperForConditionalGeneration.from_pretrained(
+        args.model, torch_dtype=torch_dtype
+    )
     model.save_pretrained(args.output)
     print(f"TRTLLM engines have been saved at {args.output}.")
+
+    model = WhisperForConditionalGeneration.from_pretrained(args.output)
+    processor = AutoProcessor.from_pretrained(
+        args.model
+    )  # TODO: save processor in args.output (like ORTModel)
+
+    data = load_dataset(
+        "hf-internal-testing/librispeech_asr_dummy", "clean", split="validation"
```

Comment:
Maybe let's use something not internal?

Reply:
This one is public, it is fine. This is the same in Transformers doc example.